5.PropTypes and DefaultProps.md

```diff
@@ -0,0 +1,40 @@
+#Prop Types And Default Props
+
+##Prop Types
+PropTypes provides **typechecking ability** to react components.
+Let's take a look at some codes.
+
+```
```

Comment:
That's actually a good point, it would be good to know if GitHub gives JSX code snippets a different styling treatment. If not let's stick with `js`.